README.md

```diff
@@ -7,26 +7,29 @@ See [this page](https://docs.prebid.org/prebid-server/overview/prebid-server-ove
 
 ## Use Maven?
 
-Easily include the Prebid Mobile SDK using Maven. Simply add this line to your gradle dependencies:
+You can easily include the Prebid Mobile SDK using Maven. If your build script is Groovy-based, add this line to your gradle dependencies:
```

Comment:
@charlie-docwriter Delete the 'easily'. It's discouraging for users who are having trouble with the instructions. For the original docs delete `easily` and `simply` and keep the rest.